site/gatsby-site/src/pages/apps/reports.js

```diff
@@ -32,6 +32,9 @@ const query = gql`
         epoch_date_downloaded
```

Comment:
these `epoch_date_` fields should no longer be requested or needed, right? or do you plan to remove them in a followup PR, along with updating the schema.json?

Reply:
I plan to remove them in a follow up otherwise it was too many dangerous changes in one PR.